src/main/java/org/symphonyoss/symphony/messageml/markdown/nodes/EmojiNode.java

```diff
@@ -68,7 +68,7 @@ public String getAnnotation() {
    */
   @Deprecated
   public void setAnnotation(String name) {
-    this.annotation = annotation;
```

Comment:
interesting :) (I guess this is not really used? or at least not covered by tests)

Reply:
yes it is marked as deprecated so I guess is not used, not anymore at least! but sonar just started complaining https://sonarcloud.io/project/issues?id=symphonyoss_messageml-utils&open=AXpDFg3mcsF6DKPN1PlJ&resolved=false&types=BUG